src/ray/core_worker/transport/direct_task_transport.h

```diff
@@ -174,11 +175,23 @@ class CoreWorkerDirectTaskSubmitter {
   absl::flat_hash_map<rpc::WorkerAddress, std::shared_ptr<rpc::CoreWorkerClientInterface>>
       client_cache_ GUARDED_BY(mu_);
 
-  /// Map from worker address to the lease client through which it should be
-  /// returned and its lease expiration time.
-  absl::flat_hash_map<rpc::WorkerAddress,
-                      std::pair<std::shared_ptr<WorkerLeaseInterface>, int64_t>>
-      worker_to_lease_client_ GUARDED_BY(mu_);
+  // max_tasks_in_flight_per_worker_ limits the number of tasks that can be pipelined to a worker using a single lease.
+  const uint32_t max_tasks_in_flight_per_worker_;
+
+  
+  
+  /// A lease_entry struct is used to condense the metadata about a single executor:
+  /// (1) The lease client through which the worker should be returned
+  /// (2) The expiration time of a worker's lease.
+  /// (3) The number of tasks that are currently in flight to the worker
+  typedef struct lease_entry {
```

Comment:
Can you follow the naming convention used [here](https://github.com/ray-project/ray/blob/master/src/ray/core_worker/transport/direct_actor_transport.h#L109) (`LeaseEntry` instead of `lease_entry` and no `typedef`)?